htmldialogelement-basic/index.html

```diff
@@ -9,25 +9,25 @@
   <!-- Simple pop-up dialog box, containing a form -->
   <dialog id="favDialog">
     <form method="dialog">
-      <section>
-        <p><label for="favAnimal">Favorite animal:</label>
+      <p>
```

Comment:
Perhaps a `<fieldset>` tag is better suited here?

Reply:
that wouldn't be useful in this example.